autogen/defaultModel.json

```diff
@@ -0,0 +1,28 @@
+{
+    "system": {
+        "target": "audiomini",
+        "sampleClockFrequency": 48000,
+        "systemClockFrequency": 98304000,
+
+        "audioIn": {
+            "wordLength": 24,
+            "fractionLength": 23,
+            "signed": true,
+            "numberOfChannels": 2
+        },
+        "audioOut": {
+            "wordLength": 24,
+            "fractionLength": 23,
+            "signed": true
```

Comment:
missing a `,` at the end of this line.

Reply:
Thanks. I wish JSON allowed trailing commas, it would make these issues much easier to avoid

Reply:
Using a json linter in your editor can help :wink: But yes, I agree and have done the same thing many times.